docs/getting-started.md

```diff
@@ -40,30 +40,67 @@ curl localhost:10101/status
+In order to better understand Pilosa's capabilities, we will create a sample project called "Star Trace" containing information about 1,000 popular Github repositories which have "go" in their name. The Star Trace index will include data points such as programming language and stargazers—people who have starred a project.
 
-#### Create the Schema
+Although Pilosa doesn't keep the data in a tabular format, we still use the terms "columns" and "rows" when describing the data model. We put the primary objects in columns, and the properties of those objects in rows. For example, the Star Trace project will contain an index called "repository" which contains columns representing Github repositories, and rows representing properties like programming languages and stargazers. We can better organize the rows by grouping them into sets called Fields. So the "repository" index might have a "languages" field as well as a "stargazers" field. You can learn more about indexes and fields in the [Data Model](../data-model/) section of the documentation.
 
-Note:
-If at any time you want to verify the data structure, you can request the schema as follows:
+<div class="note">
+ <p>If at any time you want to verify the data structure, you can request the schema as follows:<\p>
+<\div>
 
 ``` request
 curl localhost:10101/schema
 ```
 ``` response
-{"indexes":null}
+{
+  "indexes": [
+    {
+      "name": "repository",
+      "options": {
+        "keys": false,
+        "trackExistence": true
+      },
+      "fields": [
+        {
+          "name": "language",
+          "options": {
+            "type": "set",
+            "cacheType": "ranked",
+            "cacheSize": 50000,
+            "keys": false
+          }
+        },
+        {
+          "name": "stargazer",
+          "options": {
+            "type": "time",
+            "timeQuantum": "YMDH",
+            "keys": false,
+            "noStandardView": false
+          }
+        }
+      ],
+      "shardWidth": 1048576
+    }
+  ]
+}
 ```
+<div class="note">
+	<p>Note: This is the response you should receive once completing this project. It has also been formatted using `jq`. <\p>
+</div>
+
+#### Using Curl
+
+##### Create the Schema
 
-Before we can import data or run queries, we need to create our indexes and the fields within them. Let's create the repository index first:
+Before we can import data or run queries, we need to create our indexes and the fields within them. Let's create the `repository` index first:
 ``` request
 curl localhost:10101/index/repository -X POST
 ```
 ``` response
 {"success":true}
 ```
-The index name must be 64 characters or less, start with a letter, and consist only of lowercase alphanumeric characters or `_-`.
+The index name must be 64 characters or less, start with a letter, and consist only of lowercase alphanumeric characters or `_-`. The same goes for field names.
```

Comment:
This is a grammar rule that I can never remember properly, but I think the "less" here should be "fewer". If you can verify that, let's change this and other occurrences.

Reply:
You are correct. Dictionary.com says that fewer should be used for countable things while less should be used for "singular mass nouns." For example, fewer ingredients and less salt.